tests/YGDirtyMarkingTest.cpp

```diff
@@ -70,6 +70,55 @@ TEST(YogaTest, dirty_propagation_only_if_prop_changed) {
   YGNodeFreeRecursive(root);
 }
 
+TEST(Yogatest, dirty_mark_all_children_as_dirty_when_display_changes)
+{
+  const YGNodeRef root = YGNodeNew();
+  YGNodeStyleSetFlexDirection(root, YGFlexDirectionRow);
+  YGNodeStyleSetHeight(root, 100);
+  
+  const YGNodeRef tab1 = YGNodeNew();
```

Comment:
can you use similar naming for nodes as other tests? child0, child1, child1_child0 etc.